Rakefile

```diff
@@ -4,6 +4,5 @@
 require File.expand_path('../config/application', __FILE__)
 require 'coveralls/rake/task'
 
-
```

Comment:
@lasley not too sure what is wrong here, I tried switching lines of code here, but it made no difference. I still get 38 offenses, most of which are pointing that the comments in the files are too long or absence of comments at the top of some modules. Not sure if I understood what else I would need to fix? Delete these comments from files, just shorten them, or?